README.md

```diff
@@ -156,6 +162,8 @@ python scripts/load_incidents_v3.py --authz 'Token <YOUR_AUTH_TOKEN>' scripts/sa
 Note that the import process will take roughly two hours for the full data set; you can cut down the
 number of records with `head` on the individual CSVs.
 
+The `load_incidents_v3.py` script will also create a schema for you. If you already have a schema in place, and simply want to load data associated with that schema, you will need to modify the script accordingly: change the `schema_id = create_schema(...)` line with `schema_id = 'replace-this-with-the-existing-schema-id'`.
```

Comment:
If it's useful, I actually have a change I made as part of the Brazil import that adds optional parameters to the script [so you can specify the schema or record type UUID as part of the command](https://github.com/WorldBank-Transport/DRIVER/blob/23dedc5aa4e76293c99f136d427b67b4adc620b5/scripts/load_incidents_v3.py#L280-L294)

Reply:
Cool, that does sound like it'd be a useful addition.